### DISCUSSION TITLE
Booting into the original Flashforge System using REBOOT STOCK SYSTEM macro

### DISCUSSION THREAD

**Discussion by johncotten** [xblax/flashforge_ad5m_klipper_mod#67] [General]
I think this is the coolest implementation! Want to verify that it worked in Flashprint? Just reboot into the original system! Want to go back? Just power down and turn it back on!

**Comment by Giterdun2** [comment]
While that is most certainly a very cool option, it would be great to be able to connect with either Orca or Flashprint without having to reboot, like with the the original firmware. Maybe/hopefully in the final release? ;)

**Reply by consp** [reply]
> Orca 

Orca supports "normal" protocols and you can use it, flashforge slicer is the odd one out.

**Reply by xblax** [reply]
I can't really imagine anyone willing to backport that Flashforge protocol to Klipper :smile: Compared to the available open source slicers, Flashforge Slicer is a rather useless tool. I don't think it's needed for the audience of this mod.
